v4/examples/otel/go.mod

```diff
@@ -0,0 +1,12 @@
+module github.com/newrelic/go-agent/v4/examples/otel
+
+go 1.14
+
+require (
+	github.com/newrelic/go-agent/v4 v4.0.0
+	github.com/newrelic/opentelemetry-exporter-go v0.1.0
+	go.opentelemetry.io/otel v0.5.0
+	gopkg.in/yaml.v2 v2.2.7 // indirect
```

Comment:
Note to self: I assume this is brought in via the `otel` dependency, but given how we are cautious in adding additional dependencies, it's probably a good idea to make sure we understand why this is included now.